src/Tests/Installer/ThunderInstallerTest.php

```diff
@@ -162,6 +162,27 @@ public function testInstalled() {
     $this->assertResponse(200);
     // Confirm that we are logged-in after installation.
     $this->assertText($this->rootUser->getUsername());
+
+    /** @var \Drupal\Core\Database\Query\SelectInterface $query */
+    $query = \Drupal::database()->select('watchdog', 'w');
+
+    /** @var \Drupal\Core\Database\Statement $results */
+    $results = $query->fields('w', array(
```

Comment:
I'm not sure, but I think there is no need to fetch all columns (it can be only wid) or even better would be to just execute "count(*)" on database side and compare it in PHP.